server_addon/max/server/settings/main.py

```diff
@@ -4,6 +4,9 @@
 from .render_settings import (
     RenderSettingsModel, DEFAULT_RENDER_SETTINGS
 )
+from .preview_animation import (
+    PreviewAnimationModel, DEFAULT_PREVIEW_ANIMATION_SETTINGS
```

Comment:
Should we renamed this Model as well to be explicit about it belonging to CreateReview?

Reply:
I am thinking about this too, this also means the py script need to be renamed as `create_review.py`